### ISSUE TITLE
Changing case in bibliography titles

### ISSUE THREAD

**Issue by Jellby** [josephwright/achemso#27]
I need to write article titles in the bibliography, but change the case, and I can't find how to do it (if it's possible). As per the Chem. Rev. instructions: "All titles in references should have the same format - Title Case (preferred) or non-title case; mixed references formatting is not allowed".

**Comment by josephwright** [comment]
This is a tricky one: I've stayed away from case changing quite deliberately. The issue with 'title casing' is that there are value judgements (what counts as a 'significant' word), whilst for lower casing one runs into the risk of unprotected important capitals. So I've left this alone to date.

I know the ACS are asking for titles in all submissions now, so perhaps I'll need to revisit it. Certainly a flag 'force lower casing' is a possible.

**Comment by Jellby** [comment]
Of course, I wasn't thinking of changing to title case. Since they ask for consistent casing I thought I'd change everything to lowercase as the standard styles (or is it only with makebst?) do, but found out that's not an option with achemso. in my bib files I try to keep protected capitals when they matter, anyway.

Personally, i think this kind of stylistic nitpicking should be done by the editor, not by the author...

**Comment by josephwright** [comment]
@Jellby  Yes, totally agree on this not really being the author's job: but also see the need to deal with awkward editorial workflows!

**Comment by emleddin** [comment]
There's a nice solution on [TeXExchange](https://tex.stackexchange.com/a/635924) that addresses this with the [`titlecaps`](https://ctan.org/pkg/titlecaps?lang=en) package.

To get it to work as expected using the modified `.bst` and the demo documents, I made a few changes to the demo. I just added the following after the document declaration:
```
\usepackage{titlecaps}
\Addlcwords{is with of in the an from a iv v}
```

The numerals are included in the `\Addlcwords` call here because `titlecaps` parses anything inside parentheses with the first letter normally capitalized. For me, it skips the first character of the `\textsc` call (even with `\textnc{\textsc{(IV)}}`, but I'm likely misunderstanding the escape.
